spec/rest/logger.test.js

```diff
@@ -0,0 +1,139 @@
+"use strict";
+
+define(['shared_helper'], function(helper) {
+	var exports = {};
+
+	var isBrowser = !!(typeof window !== 'undefined' && typeof navigator !== 'undefined' && window.document);
+
+	/* If we are running node, we mimic the loading mechanism used in /nodejs/realtime.js and /nodejs/rest.js
+	 * to get a reference to the logger. We do this because its implementation is not a module.
+	 * 
+	 * If we are in a browser, the logger is already loaded manually via a script tag in nodeunit.html.
+	 */
+
+	var logger;
+	
+	if (isBrowser) {
+		logger = Logger;
+	} else {
+
+		var
+			fs   = require('fs'),
```

Comment:
In this repo, we should be using tabs.  Feel free to issue a separate PR to correct the space issues if you find them